app/jobs/consent_reminders_session_batch_job.rb

```diff
@@ -12,9 +12,9 @@ class ConsentRemindersSessionBatchJob < ApplicationJob
   queue_as :default
 
   def perform(session)
-    session.patients.consent_not_sent.each do |patient|
+    session.patients.without_consent.reminder_not_sent.each do |patient|
```

Comment:
It might be slightly cleaner / more encapsulated to push this logic by defining a 3rd scope on `Patient`:

```ruby
scope :needing_consent_reminder, -> { without_consent.reminder_not_sent }
```

Reply:
Great suggestion!